src/app/locales/counterpart/it.js

```diff
@@ -0,0 +1,30 @@
+// The translations in this file are added by default.
+
+'use strict';
+
+module.exports = {
+    counterpart: {
+        names: require('date-names/en'),
+        pluralize: require('pluralizers/en'),
+
+        formats: {
+            date: {
+                'default':  '%a, %e %b %Y',
+                long:       '%A, %B %o, %Y',
+                short:      '%b %e'
```

Comment:
this is incorrect. italian short date format is `%e %b.', provided `%b` is the abbreviated month name